errorhandling/src/main/kotlin/no/nav/tms/common/errorhandling/http/Exceptions.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.tms.common.errorhandling.http
+
+import io.ktor.client.plugins.*
+import io.ktor.http.*
+import no.nav.tms.common.errorhandling.logging.LoggableException
+import no.nav.tms.common.errorhandling.redactedMessage
+
+
+class ServiceRequestException(
+    service: String,
+    url: String,
+    originalThrowable: Throwable,
+    val statusCode: HttpStatusCode
+) : LoggableException(originalThrowable) {
+    constructor(service: String, url: String, clientRequestException: ClientRequestException) : this(
+        service,
+        url,
+        clientRequestException,
+        clientRequestException.response.status.tmsResponseCode()
+    )
+
+    constructor(service: String, url: String, serverResponseException: ServerResponseException) : this(
+        service,
+        url,
+        serverResponseException,
+        serverResponseException.response.status.tmsResponseCode()
+    )
+
+    init {
+        print(this)
+    }
+
+    override val summary: String =
+        """ Kall til $service med url ${url.redactedMessage(true)} feiler
+            ${stackTraceSummary(originalThrowable)}
+    """.trimIndent()
+
+    companion object {
```

Comment:
Foretrekker å beholde de opprinnelige verdiene. Heller:

`when (value) {
            in 200..299 -> HttpStatusCode.OK
            in 300..499 -> HttpStatusCode.InternalServerError
            in 500..599 -> HttpStatusCode.ServiceUnavailable
            else -> HttpStatusCode.InternalServerError
        }`